i18n/validate.py

```diff
@@ -46,7 +46,7 @@ def msgfmt_check_po_file(filename):
     """
     # Use relative paths to make output less noisy.
     rfile = os.path.relpath(filename, config.LOCALE_DIR)
-    out, err = call('msgfmt -c {}'.format(rfile), working_directory=config.LOCALE_DIR)
+    out, err = call('msgfmt -c -o /dev/null {}'.format(rfile), working_directory=config.LOCALE_DIR)
```

Comment:
We've been having trouble with failures being lost in the static pipeline due to piping to ``/dev/null``. In this case it looks okay because you're logging any errors. Might there be a need to see the full output when debugging a failure with this script? My gut instinct is to leave the output as it was, unless it is really verbose.